lib/bundler/audit/advisory.rb

```diff
@@ -136,20 +138,43 @@ def identifiers
       #   The criticality of the vulnerability based on the CVSS score.
       #
       def criticality
-        if cvss_v3
-          case cvss_v3
```

Comment:
If we're using the same logic for calculating criticality for both `cvss_v3` and `cvss_v4`, why not just combine the two?

```ruby
if cvss_v4 || cvss_v3
  case cvss_v4 || cvss_v3
  when ...
```